scale/job/configuration/configuration/job_configuration.py

```diff
@@ -288,17 +293,18 @@ def configure_logging_docker_params(self, job_exe_id):
             es_urls = ','.join(hosts)
 
         if settings.LOGGING_ADDRESS is not None:
-            self.add_pre_task_docker_param(DockerParam('log-driver', 'syslog'))
-            self.add_pre_task_docker_param(DockerParam('log-opt', 'syslog-address=%s' % settings.LOGGING_ADDRESS))
-            self.add_pre_task_docker_param(DockerParam('log-opt', 'tag=scale_%d_pre' % job_exe_id))
+            if not job_exe.is_system:
+                self.add_pre_task_docker_param(DockerParam('log-driver', 'syslog'))
```

Comment:
I changed them to take a list of Docker params and updated all of the callers. It did shorten some of the code a decent bit.